### ISSUE TITLE
program closes automatically after some time

### ISSUE THREAD

**Issue by hitman72ita** [Mygod/VPNHotspot#160]
hi, I am using/testing an old moto g first gen with custom android 9 rom as wifi repeater. VPNHotspot works well but after some time (it's random, 1 hour, 8 hours, 24 hours) when nobody is connected (I don't have devices permanently connected to it) it closes automatically. Of course I have disabled the app from battery optimization, any idea? thank you

**Comment by Mygod** [comment]
Make sure you have disabled turning off hotspot automatically in your system tethering settings.

**Comment by hitman72ita** [comment]
yes done, but the strange thing is that happens after irregular/random intervals of time, could it be useful to use an "old" android version like 7.1.2? thank you
